ext/native-decls/GetAllPeds.md

```diff
@@ -16,3 +16,36 @@ The data returned adheres to the following layout:
 
 ## Return value
 An object containing a list of peds handles.
+
+## Examples
+
+```lua
+--[[ 
+Example =>
+  Using GetAllPeds, This example prints information of all peds, Ped id, Owner, Model and Armour.
+]]
+
+-- This useful function GetPedOwner returns the player that owned the given ped.
+function GetPedOwner(ped)
+    local activePlayers = GetActivePlayers()
+    for i, player in ipairs(activePlayers) do
+        local iterPed = GetPlayerPed(player)
+        if ped == iterPed then return player end
+    end
+    return nil
+end
+
+function PrintAllPedsInformation()
```

Comment:
Theres not really a need to wrap this in a function, it just adds an extra indent for no reason